packages/frontend/src/features/sqlRunner/components/visualizations/BarChart.tsx

```diff
@@ -1,15 +1,15 @@
-import { type BarChartConfig } from '@lightdash/common';
 import EChartsReact, { type EChartsReactProps } from 'echarts-for-react';
 import { type FC } from 'react';
 import { type useSqlQueryRun } from '../../hooks/useSqlQueryRun';
+import { useAppSelector } from '../../store/hooks';
 import { useBarChartDataTransformer } from '../../transformers/useBarChartDataTransformer';
 
 type BarChartProps = {
     data: NonNullable<ReturnType<typeof useSqlQueryRun>['data']>;
-    config: BarChartConfig | undefined;
 } & Partial<Pick<EChartsReactProps, 'style'>>;
 
-const BarChart: FC<BarChartProps> = ({ data, config, style }) => {
+const BarChart: FC<BarChartProps> = ({ data, style }) => {
+    const config = useAppSelector((state) => state.barChartConfig.config);
```

Comment:
Could you move to the parent component and keep the prop `config` pls? 
So it doesn't break sql charts in dashboards (we don't have RTK in any other part of the app)

Reply:
I believe this is done